### PR TITLE
tar: added xattrs/acl support

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

**Comment by chneukirchen** [comment]
+1 if this works.

**Comment by chneukirchen** [comment]
(Still, xbps uses libarchive to extract tar.)

**Comment by thypon** [comment]
:-S, libarchive support xattrs like bsdtar, but i think it's not saved in a compat way to how GNU tar saves them. I have to check.

File: srcpkgs/tar/template [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 pkgname=tar
 version=1.28
-revision=5
+revision=6
 bootstrap=yes
 build_pie=yes
 build_style=gnu-configure
 configure_args="gl_cv_struct_dirent_d_ino=yes"
 conflicts="chroot-tar>=0"
+makedepends="attr-devel acl-devel"
```
Comment (xtraeme) [inline srcpkgs/tar/template]: acl-devel depends on attr-devel, so that attr-devel is redundant.

**Pull request merged**